assignments/arrays.js

```diff
@@ -63,32 +65,81 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+// console.log(inventory[32])
+// console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*`);
+
+for(let i = 0; i < inventory.length; i++){
+  if(inventory[i].id === 33) {
+    //string.length = 50
+    //Foreach iteration: i = ?      i < inventory.length   i++   if(inventory[i].id === 33)           
+    //1st                   0                         0 < 50 true           1      1 === 33 false                        
+    //2nd                 1                          1 < 50 true           2      2 === 33 false                        
```

Comment:
Excellent job analyzing first how you would tackle the problem before really coding it! 💯 🥇